specification-release/apigee_client.py

```diff
@@ -0,0 +1,164 @@
+import requests

```

Comment:
This file isn't needed, it's provided by https://github.com/NHSDigital/api-management-utils during release

Reply:
removed